src/components/FormFields/index.js

```diff
@@ -104,39 +108,50 @@ class FormFields extends React.Component {
         let startTime
         let endTime
         let subEventKeys = Object.keys(this.props.editor.values.sub_events)
-        let key = subEventKeys.length > 0 ? Math.max.apply(null, subEventKeys)+1 : 1
+        let key = subEventKeys.length > 0 ? Math.max.apply(null, subEventKeys)+1 : 0
         if (_.keys(this.props.editor.values.sub_events).length) {
             const subEvents = this.props.editor.values.sub_events
             const startDates = []
             const endDates = []
             for (const key in subEvents) {
-                if (subEvents.hasOwnProperty(key)) {
+                if (subEvents.hasOwnProperty(key) && subEvents[key].start_time !== undefined && subEvents[key].end_time !== undefined) {
```

Comment:
to keep your code cleaner, you can:
`if (subEvents.hasOwnProperty(key) && subEvents[key].start_time && subEvents[key].end_time ) {`
result will be the same (you can try it on console)